docs/modules/ROOT/pages/how-tos/testing/integration/group-snapshots.adoc

```diff
@@ -0,0 +1,75 @@
+= Creating a group snapshot
+
+In {ProductName}, the integration service provides `group` snapshot to allow users to submit multiple related pull/merge requests under one application to their repos, have them tested together and have the CI results posted back to pull/merge requests.
+
+== Procedures
+To have `group` snapshot created for multiple pull/merge requests, complete the following steps:
+
+. Identify the application and components that are related and need to be updated.
+. Navigate to their git repos.
+. Create branches with the same branch name for all of these git repos.
+. Commit code changes to branches created above and create pull/merge requests.
+
+After every build pipelinerun succeeds and component snapshot is created, group snapshot will be created for the new built component container images and other components. And then integration test runs for component and group snapshot.
```

Comment:
Would something along these lines work? Does it improve clarity?
```suggestion
After every build pipelinerun succeeds and component snapshot is created, a group snapshot will be created containing the latest build for all components which are grouped by the branch names. The configured ItegrationTestScenario(s) will then be run with the group snapshot.
```

Reply:
Updated. Thanks.